docs/explainer.md

```diff
@@ -160,4 +160,38 @@ let writableStream = tcpSocket.writable;
 tcpSocket.close();
 ```
 
+To simplify security analysis, an API for listening for incoming connections is not yet being proposed.
+
 ## UDP
+
+To simplify security analysis, the initial proposal only supports cases where the web app initiates communication with a remote host.
+
+```
+const options = {
+    remoteAddress: 'example.com',
+    remotePort: 7
+};
+
+try {
+  const udpSocket = await navigator.openUDPSocket(options);
+  doStuffWith(udpSocket);
+} catch (err) {
+  // handle error
+}
+```
+
+`send` returns a promise, that resolves when a message is sent:
+
+```
+let blob = ...;
+await udpSocket.send(blob);
```

Comment:
If we later allow a single UDP socket to communicate with multiple end points, `send` can accept a second argument - a dictionary containing remoteAddress and remotePort.